compile/compile.go

```diff
@@ -437,6 +437,10 @@ func (c *Compiler) compileWasm(ctx context.Context) error {
 
 	// Each entrypoint needs an entry in the manifest along with the
 	// original rule(s) removed from the remaining rego modules.
+
+	// For each package path keep a list of new imports. They are stored as a
```

Comment:
I think this logic is complex enough that we should factor it into a separate function.

Reply:
If there are multiple entrypoints in the same package the output will be non-deterministic since the imports are stored in a map. I could imagine just refactoring this to be map[string][]*ast.Import where the key is the package. The code below that adds the required imports can just search the slice to see if the import already exists (the slice should be small in practice).

Reply:
We should have a test case covering this...

Something like:

module1.rego

```
package test

p { q }
```

module2.rego:

```
package test

q { r }
r { s }
s = 7
```

Compile out entrypoints `q` and `s`. The result should be:

module1.rego:

```
package test

import data.test.q
import data.test.s

p { q }
```

module2.rego

```
package test

import data.test.q
import data.test.s

r { s }
```